src/script/build-integration-branch

```diff
@@ -150,6 +177,12 @@ def main():
                 message = ('Exiting due to an unknown failure when pulling '
                            f'PR#{pr_number}')
                 raise Exception(message)
+    if cli.trailers:
+        cmd = ['git', 'commit', '--am', '--no-edit']
```

Comment:
I had no idea one could abbreviate the option name, so was wondering what --am was